src/routes/manage-selections/ManageSelectionsScreen.tsx

```diff
@@ -24,11 +24,17 @@ const Root = styled.ScrollView`
 
 const SectionHeading = styled(Subheading)`
   padding-top: ${props => props.theme.spacings.xl};
+  font-family: ${props => props.theme.fonts.contentFontBold};
+  color: ${props => props.theme.colors.text};
 `
 
 const Padding = styled.View`
   padding-bottom: ${props => props.theme.spacings.xxl};
 `
+const StyledHeading = styled(Heading)`
+  text-align: center;
+  color: ${props => props.theme.colors.text};
```

Comment:
Please use the `HeadingText` component for inheritance, instead of the `Heading` component, then you can remove the `color` here.